src/TwirpRPC.ts

```diff
@@ -24,7 +24,7 @@ export class TwirpRpc {
 
   constructor(host: string, pkg: string, prefix?: string) {
     if (host.startsWith('ws')) {
-      host = host.replace('ws', 'http');
+      host = host.replace(/^ws/, 'http');
```

Comment:
lg! 
not sure if this is actually needed, as we check for `startsWith` already before and `string.replace` only changes the first occurrence.

Reply:
yeah you are right.